compact.go

```diff
@@ -607,7 +608,7 @@ func (c *LeveledCompactor) write(dest string, meta *BlockMeta, blocks ...BlockRe
 	}
 
 	// Create an empty tombstones file.
-	if _, err := writeTombstoneFile(c.logger, tmp, newMemTombstones()); err != nil {
+	if _, err := tombstones.WriteTombstoneFile(c.logger, tmp, tombstones.NewMemTombstones()); err != nil {
```

Comment:
Are we sure `tombstones.WriteTombstoneFile` needs to be `WriteTombstoneFile` etc? Why not just `tombstones.WriteFile` if we have clear package name?

Reply:
same for `NewMemTombstones`